src/CMakeLists.txt

```diff
@@ -35,6 +35,94 @@ add_library(llama
             unicode-data.cpp
             unicode.cpp
             unicode.h
+            models/llm_build_arcee.cpp
+            models/llm_build_arctic.cpp
+            models/llm_build_arwkv7.cpp
+            models/llm_build_baichuan.cpp
+            models/llm_build_bailingmoe.cpp
+            models/llm_build_bert.cpp
```

Comment:
Just want to remind this from [CONTRIBUTING.md](https://github.com/ggml-org/llama.cpp/blob/master/CONTRIBUTING.md)

> Create separate PRs for each feature or fix. Avoid combining unrelated changes in a single PR